services/twilioService.js

```diff
@@ -148,24 +122,15 @@ class TwilioService {
 
   getChannelName(contact) {
     const localToken = JSON.parse(this.hmToken);
-    return localToken._id > contact ? `${localToken._id}.${contact}` : `${contact}.${localToken._id}`;
+    return localToken._id > contact.id ? `${localToken._id}.${contact.id}` : `${contact.id}.${localToken._id}`;
   }
 
   async updateMessages(contact) {
-    const messages = await this.getRawMessages(contact);
+    const channel = await this.chatClient.getChannelBySid(contact.channelSid);
+    const messages = await channel.getMessages(50);
```

Comment:
Can we add a constant for the 50

Reply:
that was actually forgotten from testing, I'd rather not have any constant and revert to Twilio's default. I pushed a commit that changes this, please let me know if you'd like to set a constant

Reply:
I'm happy with the change. Thanks Bassam!